FlashFeetAPI/src/main/java/org/example/api/AuthService.java

```diff
@@ -0,0 +1,55 @@
+package org.example.api;
+
+
+import org.example.cli.Login;
+import org.example.client.FailedToVerifyTokenException;
+import org.example.db.AuthDao;
+import org.example.db.DatabaseConnector;
+import org.example.exception.DatabaseConnectionException;
+import org.example.exception.FailedToLoginException;
+import org.example.exception.FailedTogenerateTokenException;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+public class AuthService {
+    private AuthDao authDao = new AuthDao();
+    private DatabaseConnector databaseConnector = new DatabaseConnector();
+    Connection c = databaseConnector.getConnection();
+
+    public AuthService() throws DatabaseConnectionException, SQLException {
+
+    }
+
+    public AuthService(AuthDao authDao, DatabaseConnector databaseConnector) throws DatabaseConnectionException, SQLException {
+
+    }
```

Comment:
```suggestion
    private AuthDao authDao;
    Connection c;

    public AuthService(AuthDao authDao, DatabaseConnector databaseConnector) throws DatabaseConnectionException, SQLException {
this.authDao = authDao;
this.c = databaseConnector.getConnection();
    }
```

Reply:
you're not setting the parameters being passed in to your constructor so you wouldn't be able to mock them out in tests